projects/onto-search/src/lib/onto-search-facet/dropdown-multi-select-facet/dropdown-multi-select-facet.component.ts

```diff
@@ -0,0 +1,70 @@
+import {Component, Input, TemplateRef} from '@angular/core';
+import {OntoSearchFacetComponent} from '../onto-search-facet.component';
+import {FormControl} from '@angular/forms';
+import {ReplaySubject} from 'rxjs';
+import {SearchFacetType} from '../models/search-facet-type';
+import {SearchFacetModel} from '../models/search-facet-model';
+import {untilComponentDestroyed} from '@w11k/ngx-componentdestroyed';
+import {MatSelectChange} from '@angular/material/select';
+
+@Component({
+  selector: 'onto-dropdown-multi-select-facet',
+  templateUrl: './dropdown-multi-select-facet.component.html',
+  styleUrls: ['./dropdown-multi-select-facet.component.scss']
+})
+export class DropdownMultiSelectFacetComponent extends OntoSearchFacetComponent {
+  @Input()
+  public facetTemplate: TemplateRef<any>;
+
+  @Input()
+  public facetTitleTemplate: TemplateRef<any>;
+
+  @Input()
+  public type: SearchFacetType;
+
+  /** control for the selected from multi-selection */
+  public selectedFormControl: FormControl = new FormControl();
+
+  /** control for the MatSelect filter keyword multi-selection */
+  public filtredFormControl: FormControl = new FormControl();
+
+  /** list of filtered by search keyword for multi-selection */
+  public filtered: ReplaySubject<SearchFacetModel[]> = new ReplaySubject<SearchFacetModel[]>(1);
+
+  constructor() {
+    super();
+  }
+
+  ngOnInit(): void {
+    this.filtered.next(this.data.facetGroupData.slice());
```

Comment:
Everywhere you use `this.data`. I think it is better to keep the input decoupled from the implementations. In the parent facet on init, we set internal variables. Why not use those?